tests/connections/test_unix_socket_connection.py

```diff
@@ -47,9 +48,9 @@ class ThreadedUnixStreamServer(
 
 
 class UnixSocketConnectionTestCase(unittest.TestCase):
-    # pylint: disable=protected-access
+    # pylint: disable=protected-access, invalid-name
     def setUp(self):
-        self.socketname = "%s/%s.sock" % (
+        self.socketname = "{}/{}.sock".format(
```

Comment:
I think we should use f-strings now for changes string formatting.

Reply:
Maybe. But this again means: f-String everywhere (so better for another PR).